src/util.ts

```diff
@@ -56,9 +59,9 @@ export async function waitForNode(
   waitTimeout: number,
 ): Promise<void> {
   const { Runtime } = client
```

Comment:
when running ava under nyc, it is added with coverage code which breaks once passed to the browser (i.e cov_### globals)